app/representers/api/v1/user_representer.rb

```diff
@@ -50,10 +50,10 @@ class UserRepresenter < Roar::Decorator
              readable: true,
              writeable: false
 
-    property :support_identifier,
-             type: String,
+    property :consent_preferences,
+             type: JSON,
```

Comment:
I'm not totally sure if this is the right way to do this (but it does seem to work)